### PR TITLE
Update test_accelerator_connector.py

### PR DESCRIPTION
**Pull request by justusschock**
## What does this PR do?

Part of #10169

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: tests/accelerators/test_accelerator_connector.py [hunk 1]
```diff
@@ -463,7 +469,7 @@ def test_ipython_compatible_backend(*_):
 @pytest.mark.parametrize(["accelerator", "plugin"], [("ddp_spawn", "ddp_sharded"), (None, "ddp_sharded")])
 def test_plugin_accelerator_choice(accelerator: Optional[str], plugin: str):
     """Ensure that when a plugin and accelerator is passed in, that the plugin takes precedent."""
-    trainer = Trainer(accelerator=accelerator, plugins=plugin, num_processes=2)
+    trainer = Trainer(accelerator=strategy, plugins=plugin, num_processes=2)
```
Comment (four4fish) [inline tests/accelerators/test_accelerator_connector.py:472]: strategy=accelerator?

File: tests/accelerators/test_accelerator_connector.py [hunk 2]
```diff
@@ -463,7 +469,7 @@ def test_ipython_compatible_backend(*_):
 @pytest.mark.parametrize(["accelerator", "plugin"], [("ddp_spawn", "ddp_sharded"), (None, "ddp_sharded")])
```
Comment (four4fish) [inline tests/accelerators/test_accelerator_connector.py:469]:
```suggestion
@pytest.mark.parametrize(["strategy", "plugin"], [("ddp_spawn", "ddp_sharded"), (None, "ddp_sharded")])
```

**Comment by tchaton** [comment]
@kaushikb11 Mind checking this one.

**Review by tchaton — APPROVED** [review]
LGTM !

**Comment by carmocca** [comment]
@justusschock is this still relevant?
